File: intervention-service/src/main/java/net/elghz/interventionservice/model/Equipement.java
package net.elghz.interventionservice.model;

import lombok.Data;
import net.elghz.interventionservice.enumeration.Statut;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;
@Data
public class Equipement {


    private Long id;
    private String numeroSerie;
    private String nom;
    private String descreption;
    private String type;
    private String marque;
    private Statut statut;
    private Date dateMiseService;
    private Date dateMiseHorsService;
    private List<PhotoEquipementDTO> photoEquipements = new ArrayList<>();
    private salleDTO salle;
    private typeEquipementDTO typeEquipementt;
}

File: frontEnd-Angular/src/app/models/equipement.ts
import {Salle} from "./salle";
import {TypeEquipement} from "./typeEquipement";
import {Statut} from "./statut";

export interface Equipement{
  id: number;
  code:String;
  numeroSerie: string;
  nom: string;
  descreption: string;
  type: string;
  marque: string;
  statut: string;
  dateMiseService: Date;
  dateMiseHorsService: Date;
  salle:Salle;
  typeEquipementt: TypeEquipement;
}
